OpenEdXMobile/src/test/java/org/edx/mobile/util/WhatsNewUtilTest.java

```diff
@@ -68,9 +61,10 @@ public void validateWhatsNewMessages(List<WhatsNewModel> messages) {
 
             final List<String> platforms = message.getPlatforms();
             assertThat(platforms).isNotEmpty();
+
+            // Check platform name is valid (either ios or android)
```

Comment:
`(either ios or android)` is an implementation detail, which shouldn't be mentioned here.